src/main/python/deep_qa/training/text_trainer.py

```diff
@@ -183,15 +182,15 @@ def _load_layers(self):
                         self.embedding_layers[embedding_name] = (None, layer)
                     else:
                         self.embedding_layers[embedding_name] = (layer, None)
-            elif layer.name == "sentence_encoder":
-                logger.info("  Found sentence encoder")
-                self.sentence_encoder_layer = layer
-            elif layer.name == "timedist_sentence_encoder":
-                logger.info("  Found sentence encoder")
-                if self.sentence_encoder_layer is None:
-                    self.sentence_encoder_layer = layer
-                else:
-                    logger.warning("  FOUND DUPLICATE SENTENCE ENCODER LAYER!  NOT SURE WHAT TO DO!")
+            if '_sentence_encoder' in layer.name:
```

Comment:
having just `encoder` would also pick up `seq2seq_encoder`, so i moved this check after `seq2seq_encoder` and made it an `elif`.